index.html

```diff
@@ -14,8 +14,13 @@
           <div class="logo">
           <a class="header-brand" href="/">Fadahunsi</a>
         </div>
-          <div>
-            <img src="./img/Union.png" alt="The hamburger menu" class="hamburger-icon" />
+        <div class="menu-wrapper" id="menuWrapper" onclick="menuMob()">
+          <div class="menu-overlay"></div>
+        <div class="menu-content">
+          <div class="menu-toggle" id="menuToggle">
+            <span></span>
+            <span></span>
+            <span></span>
```

Comment:
- [ ] Kindly add some padding between every option of the menu to match the original design.
- [ ] Kindly decrease the space between the close button and the end of the page, to match the original design.

Your design
![image](https://user-images.githubusercontent.com/35535130/173529367-1f4f9960-e8b2-4d36-b44a-21b2fe90c343.png)

Original design
![image](https://user-images.githubusercontent.com/35535130/173529480-0b51e6cc-f09e-46ed-b18e-52f98fce0547.png)